notebooks/multiclass/base.ipynb

```diff
@@ -0,0 +1,284 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "0b684d1c-49d8-4a89-80ee-8f54c4f6fe34",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import os\n",
+    "\n",
+    "while \"notebooks\" in os.getcwd():\n",
+    "    os.chdir(\"..\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "8f020349-c880-426d-a878-3089c79c9203",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/mbrzozowski/projects/belt_add_multiclass/.venv/lib/python3.9/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+      "  from .autonotebook import tqdm as notebook_tqdm\n"
+     ]
+    }
+   ],
+   "source": [
+    "from datasets import load_dataset\n",
+    "import numpy as np\n",
+    "\n",
+    "from belt_nlp.bert_classifier_truncated import BertClassifierTruncated"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "082da124-c5d5-4ed1-8828-fc7f92efbba2",
+   "metadata": {},
+   "source": [
+    "# Example - Model BERT with truncation of longer texts"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "bebcd04f-9a46-4562-8afd-25e0f41263ca",
+   "metadata": {},
+   "source": [
+    "In this notebook we will show how to use basic methods `fit` and `predict` for the BERT model with truncating texts longer than 512 tokens."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "77184f93-d4e0-449f-8a5e-c9b1103d077c",
+   "metadata": {},
+   "source": [
+    "## Load data - author recognition of Guardian articles"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "b8990378",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = load_dataset(\"guardian_authorship\",\"cross_topic_1\")"
```

Comment:
A space is missing after the comma.

CI is not set up yet, but for now, you can check the errors with `nbqa flake8 .`